src/components/base/outline-google-map/outline-google-map.stories.ts

```diff
@@ -0,0 +1,202 @@
+        '**OPTIONAL Number:** Sets the starting map longitude. FitToMarkers overrides.',
+    },
+    zoom: {
+      description:
+        '**OPTIONAL Number:** Sets the starting map zoom level. FitToMarkers overrides on inital page load, adjusting in Storybook will override FitToMarkers.',
+    },
+    fitToMarkers: {
+      description:
+        '**Boolean**: Sets the starting map zoom level to show all markers. Overrides zoom and center.',
+    },
+    mapType: {
+      description:
+        '**OPTIONAL MapType** Sets the style of map ("roadmap", "satellite", "hybrid", "terrain").',
+      options: mapTypeOptions,
+      control: {
+        type: 'select',
+      },
+    },
+    gestureHandling: {
+      description:
+        '**GestureType** Sets how the map handles touch events ("auto", "cooperative", "greedy", "none").',
+      options: gestureOptions,
+      control: {
+        type: 'select',
+      },
+    },
+    disableDefaultUI: {
+      description: '**Boolean**: Clears the map controls.',
+    },
+    zoomControl: {
+      description:
+        '**Boolean**: Individually adds zoom controls to map after disabling controls.',
+    },
+    mapTypeControl: {
+      description:
+        '**Boolean**: Individually adds map type controls to map after disabling controls.',
+    },
+    scaleControl: {
+      description:
+        '**Boolean**: Individually adds scale controls to map after disabling controls.',
+    },
+    streetViewControl: {
+      description:
+        '**Boolean**: Individually adds street view controls to map after disabling controls.',
+    },
+    rotateControl: {
+      description:
+        '**Boolean**: Individually adds rotation controls to map after disabling controls.',
+    },
+    fullScreenControl: {
+      description:
+        '**Boolean**: Individually adds full screen controls to map after disabling controls.',
+    },
+  },
+  parameters: {
+    viewMode: 'docs',
+    previewTabs: {
+      canvas: {
+        hidden: false,
+      },
+    },
+    docs: {
+      description: {
+        component: `This component renders a Google Map.`,
+      },
+      source: {
+        code: `
+<outline-google-map
+  api-key="{{ apiKey }}"
+  map-type="{{ mapTypeId }}"
+  center-latitude="{{ mapStartingLat }}"
+  center-longitude="{{ mapStartingLong }}"
+  fit-to-markers
+  disable-ui
+>
+  <outline-map-marker
+    slot="markers" 
+    latitude="{{ locationLat }}" 
+    longitude="{{ locationLong }}"
+    icon="{{ pngUrlorOutlineSVG }}"
+    iconFillColor="{{ htmlColororOutlineVar }}">
+      {{ mapMarkerDefaultSlot }}
+  </outline-map-marker>
+</outline-google-map>
+        `,
+      },
+    },
+  },
+};
+
+const Template = ({
+  apiKey,
+  zoom,
+  mapType,
+  centerLatitude,
+  centerLongitude,
+  fitToMarkers,
+  disableDefaultUI,
+  zoomControl,
+  mapTypeControl,
+  scaleControl,
+  streetViewControl,
+  rotateControl,
+  fullScreenControl,
+  gestureHandling,
+  defaultSlot,
+}): TemplateResult => {
```

Comment:
Minor:

I usually use this pattern for stories since listing all the variables just means I have to track them in more than one place.

```javascript
const Template = (args = configuration.args): TemplateResult => {
  args = {
    ...configuration.args,
    ...args,
  };

  return html``;
};
```

Then you can access `args.myAttribute`.